php.ini-development

```diff
@@ -442,11 +442,11 @@ memory_limit = 128M
 ; operators. The error level constants are below here for convenience as well as
 ; some common settings and their meanings.
 ; By default, PHP is set to take action on all errors, notices and warnings EXCEPT
-; those related to E_NOTICE, which together cover best practices and
-; recommended coding standards in PHP. For performance reasons, this is the
-; recommend error reporting setting. Your production server shouldn't be wasting
-; resources complaining about best practices and coding standards. That's what
-; development servers and development settings are for.
+; those related to E_NOTICE, which covers best practices and recommended coding
+; standards in PHP. For performance reasons, this is the recommended error
+; reporting setting. Your production server shouldn't be wasting resources
+; complaining about best practices and coding standards. That's what develoment
+; servers and development settings are for.
```

Comment:
I'm not even sure that is the case any more, pretty sure since 8.0.0 it is _all_ errors including `E_NOTICE` that are emitted.